django_socio_grpc/tests/fakeapp/serializers.py

```diff
@@ -91,6 +91,14 @@ class Meta:
         fields = "__all__"
 
 
+class SimpleRelatedFieldModelSerializer(proto_serializers.ModelProtoSerializer):
```

Comment:
miss comment to explain that it use a field that should use the _ConvertScalarFieldValue functionnality